### PR TITLE
Add a mimic helper that will make requests to the mimic control plane

### PR DESCRIPTION
**Pull request by cyli**
In this example, it just changes the statuses of some servers.

### CODE REVIEW

File: otter/integration/lib/test_nova.py [hunk 1]
```diff
@@ -5,14 +5,45 @@
...
 
 
-class _Response(object):
+class Response(object):
     """Fake response object"""
     def __init__(self, code):
         self.code = code
 
 
+def get_fake_treq(test_case, method, url, expected_args_and_kwargs, response):
```
Comment (manishtomar) [inline otter/integration/lib/test_nova.py]: Non-blocking: Maybe this or `StubTreq` should goto `treq` package to avoid test duplication?

Comment (cyli) [inline otter/integration/lib/test_nova.py]: Actually that's a really great idea.  It'd be nice if all side-effect-y libraries had a stub.  :)

Comment (manishtomar) [inline otter/integration/lib/test_nova.py]: Yes! All the libraries should provide their memory/stub version.

Comment (cyli) [inline otter/integration/lib/test_nova.py]: https://github.com/rackerlabs/otter/issues/1423

File: otter/integration/lib/mimic.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    """
+    Class that handles HTTP requests to the mimic nova control plane.
+    """
+    def change_server_statuses(self, rcs, ids_to_status):
+        """
+        Change the statuses of the given server IDs.
+
+        :param rcs: A :class:`otter.integration.lib.resources.TestResources`
+            instance.
+        :param dict ids_to_status: A mapping of server IDs to the string status
```
Comment (kivattik) [inline otter/integration/lib/mimic.py]: Non-blocking: Should we include a list of the possible status values here, or just reference the documentation?

Comment (cyli) [inline otter/integration/lib/mimic.py]: I added a link to the nova docs, thanks!  :)

**Comment by manishtomar** [comment]
LGTM :)

**Pull request merged**